reconcile/aus/advanced_upgrade_service.py

```diff
@@ -306,6 +306,7 @@ class ClusterUpgradePolicyLabelSet(BaseModel):
     schedule: str = Field(alias=aus_label_key("schedule"))
     mutexes: Optional[CSV] = Field(alias=aus_label_key("mutexes"))
     sector: Optional[str] = Field(alias=aus_label_key("sector"))
+    frozen: Optional[bool] = Field(alias=aus_label_key("frozen"))
```

Comment:
Did you consider using `disable_scheduling` or just `disabled` instead of frozen? Frozen leaves some room for interpretation IMHO.

Reply:
I.e. will soakdays be updated if frozen? What about inheritance in that case?

Reply:
soak days will be updated. this condition is evaluated the same as the others. condition checking happens within the context of `calculate_diff` while soak days are updated before when the version history is prepared.

i've not considered other naming. can you elaborate on other interpretations?

Reply:
Just the immutability of the policy. I.e. would frozen make the policy immutable?

Reply:
ah understood. @patjlm wdyt about `disabled`?

Reply:
I think the same mis-interpretations can occur (upgrades are disabled, not the policy, nor soakdays reports), but `disabled`  is definitely fine, probably better than `frozen`.